components/message_with_additional_content.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {Posts} from 'mattermost-redux/constants';
+
+import PostBodyAdditionalContent from 'components/post_view/post_body_additional_content';
+import PostMessageContainer from 'components/post_view/post_message_view';
+
+export default function MessageWithAdditionalContent({post, previewCollapsed, previewEnabled, isEmbedVisible}) {
```

Comment:
Nice. As parent component has the checks for update stateless serves the purpose!